File: src/main/java/com/cuupa/mailprocessor/SpringBootMailprocessorApplication.java
package com.cuupa.mailprocessor;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.web.servlet.support.SpringBootServletInitializer;

@SpringBootApplication
public class SpringBootMailprocessorApplication extends SpringBootServletInitializer {

    public static void main(String[] args) {
        SpringApplication.run(SpringBootMailprocessorApplication.class, args);
    }

}

File: src/main/kotlin/com/cuupa/mailprocessor/SpringBootMailprocessorApplication.kt
package com.cuupa.mailprocessor

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.web.servlet.support.SpringBootServletInitializer

@SpringBootApplication
open class SpringBootMailprocessorApplication : SpringBootServletInitializer() {

    companion object {
        @JvmStatic
        fun main(args: Array<String>) {
            SpringApplication.run(SpringBootMailprocessorApplication::class.java, *args)
        }
    }
}